smart_emailApp/views.py

```diff
@@ -90,15 +89,20 @@ def scheduled_email(request):
 
 
 def create_email(request):
-    form = CreateEmailForm(request.POST, request.FILES)
     if request.method == 'POST':
+        form = CreateEmailForm(request.POST, request.FILES)
         if form.is_valid():
-            # handle_uploaded_file(request.FILES["Stage_image"])
             instance = form.save(commit=False)
+            # Save the product images
+            image = form.cleaned_data.get('image')
```

Comment:
I didnt know that what I had to do, I never had to this, but if it works that fine.